modules/benchmark/R/align_data.R

```diff
@@ -4,16 +4,14 @@
 ##' @param model.calc data.frame
 ##' @param obvs.calc data.frame
 ##' @param var data.frame
-##' @param start_year numeric
-##' @param end_year numeric
 ##' @return dat
 
 ##' @author Betsy Cowdery
 
 
 ## Align timeseries data using different functions
 
-align_data <- function(model.calc, obvs.calc, var, start_year, end_year, align_method = "match_timestep") {
+align_data <- function(model.calc, obvs.calc, var, align_method = "match_timestep") {
```

Comment:
I did this grep before I saw istfer's comment, so here are the places I see that could potentially need changing (obviously irrelevant grep hits removed):

```
pecan/documentation/tutorials/AnalyzeOutput/modelVSdata.Rmd:123:aligned_dat = PEcAn.benchmark::align_data(model.calc = model, obvs.calc= observations, var ="NEE", start_year =start_year, end_year = end_year, align_method ="mean_over_larger_timestep")
pecan/modules/assim.batch/R/pda.get.model.output.R:114:    dat <- PEcAn.benchmark::align_data(model.calc = model, obvs.calc = inputs[[k]]$data, var = data.var, 
pecan/modules/benchmark/man/align_data.Rd:7:align_data(model.calc, obvs.calc, var, start_year, end_year,
pecan/modules/benchmark/R/calc_metrics.R:18:  dat <- align_data(model.calc, obvs.calc, var, start_year, end_year, 
```